centreon/packages/ui/src/InputField/Search/PersistentTooltip.tsx

```diff
@@ -58,6 +59,10 @@ const PersistentTooltip = ({
     closeTooltipProp?.();
   };
 
+  useEffect(() => {
+    setOpen(openTooltip as boolean);
+  }, [openTooltip]);
```

Comment:
I'm not really a fan of useEffect. You can remove this by just using and remove the `open` state.
```
const open = openTooltip;
```